7zip/README.md

```diff
@@ -28,80 +28,21 @@ hab pkg install core/7zip --binlink
 7z --help
 ```
 
-## Tests
+# Testing
 
-### Linux
-
-To run tests of an already built package in a new studio instance:
+For the linux plan:
 
 ```bash
-source results/last_build.env
+hab pkg build 7zip
+source ./results/last_build.env
 hab studio run "./7zip/tests/test.sh $pkg_ident"
 ```
 
-Sample output:
-
-```bash
-➜  core-plans git:(7zip) ✗ hab studio run "./7zip/tests/test.sh $pkg_ident"
-   hab-studio: Importing 'core' secret origin key
-» Importing origin key from standard input
-★ Imported secret origin key core-20190311142149.
-   hab-studio: Importing 'core' public origin key
-» Importing origin key from standard input
-★ Imported public origin key core-20190311142149.
-   hab-studio: Exported: HAB_ORIGIN=core
-» Installing core/bats
-☁ Determining latest version of core/bats in the 'stable' channel
-☛ Verifying core/bats/0.4.0/20190115015448
-...
-✓ Installed core/bats/0.4.0/20190115015448
-★ Install of core/bats/0.4.0/20190115015448 complete with 1 new packages installed.
-» Binlinking bats from core/bats/0.4.0/20190115015448 into /bin
-★ Binlinked bats from core/bats/0.4.0/20190115015448 to /bin/bats
-» Installing core/7zip/16.02/20190513161338
-☛ Verifying core/7zip/16.02/20190513161338
-...
-✓ Installed core/7zip/16.02/20190513161338
-★ Install of core/7zip/16.02/20190513161338 complete with 1 new packages installed.
-1..5
-ok 1 package directory for package ident core/7zip/16.02/20190513161338 exists
-ok 2 7z exe runs
-ok 3 7z exe output mentions expected version 16.02
-ok 4 7za exe runs
-ok 5 7za exe output mentions expected version 16.02
-```
-
-### Windows
-
-To run tests of an already built package in a new studio instance:
+For the windows plan:
 
 ```powershell
+hab pkg build 7zip
 . .\results\last_build.ps1
-hab studio run "& ./7zip/tests/test.ps1 $pkg_ident"
-```
-
-Sample output:
-
-```powershell
-PS C:\projects\core-plans> hab studio run "& ./7zip/tests/test.ps1 $pkg_ident"
-
-Id     Name            PSJobTypeName   State         HasMoreData     Location             Command
---     ----            -------------   -----         -----------     --------             -------
-1      Job1            BackgroundJob   Running       True            localhost            Microsoft.PowerShell.M...
-core/7zip/16.04/20190513101258
-» Installing core/7zip/16.04/20190513101258
-→ Using core/7zip/16.04/20190513101258
-≡ Install of core/7zip/16.04/20190513101258 complete with 0 new packages installed.
-Executing all tests in 'C:\projects\core-plans\7zip\tests/test.pester.ps1'
-
-Executing script C:\projects\core-plans\7zip\tests/test.pester.ps1
-
-  Describing The 7z bin
-
-    Context 7z invoked without options
-      [+] Runs and exits successfully 278ms
-      [+] Mentions the expected version number on stdout 207ms
-Tests completed in 1.03s
-Tests Passed: 2, Failed: 0, Skipped: 0, Pending: 0, Inconclusive: 0
-
+hab studio run -D "hab pkg install results/$pkg_artifact"
```

Comment:
Hi @mwrock.  On Windows 2016 I found that the ``hab studio run -D "hab pkg install results/$pkg_artifact"`` was required; otherwise the docker windows studio failed to find the freshly build artifact.  My observation was that on mac/linux after building a plan, the new package could be loaded immediately via the $pkg_ident. However, on Windows the artifact $pkg_artifact had to be first loaded before the installation via the $pkg_ident is successfull.

I've raised an [issue](https://github.com/habitat-sh/habitat/issues/6681) in case this is something that needs corrected.

Let me know if you are ok with the observation above so that I can merge this into master.  Thanks!

Reply:
oh that is right. you need to build the artifact. I still think its overkill to add the `-D` here but I can make peace with that. Disregard the above.

Reply:
Thanks @mwrock.  I'll resolve this conversation then.